### DISCUSSION TITLE
Fluent UI DatePicker setting value

### DISCUSSION THREAD

**Discussion by murcinar** [microsoft/fluentui#18932] [Q&A] [answered]
Hi,
I am using a Fluent UI DatePicker control where it will have no value set to it in the beginning but once another control is clicked, I will need to populate a valid date into the DatePicker and make it available to be adjusted. In the event handler where the data is loaded, I will need to be able to set the date value to the picker, where I have tried many ways including all below without any luck:
document.getElementById('startDate').setAttribute('setDate',currentNotification.startDateUTC.toString());
document.getElementById('startDate').nodeValue = currentNotification.startDateUTC.toString();
document.getElementById('startDate').setAttribute('setDate',currentNotification.startDateUTC.toString());
document.getElementById('startDate').setAttribute('value',currentNotification.startDateUTC.toString());

Can you please help me how ca I set the date on a Fluent UI Date Picker remotely?
Thanks!

**Comment by micahgodbolt** [comment, marked as answer]
There are no public methods for modifying the date picker. You'll need to specify the value via props and state.
